python/pylibcudf/pylibcudf/table.pyx

```diff
@@ -77,6 +76,31 @@ cdef class Table:
             for i in range(tv.num_columns())
         ])
 
+    # Ideally this function would simply be handled via a fused type in
+    # from_table_view, but this does not work due to
+    # https://github.com/cython/cython/issues/6740
+    @staticmethod
+    cdef Table from_table_view_of_arbitrary(const table_view& tv, object owner):
```

Comment:
Do you think this API will help with this Morpheus use case of creating a cuDF Python object from a `table_view`? https://github.com/nv-morpheus/Morpheus/blob/branch-25.06/python/morpheus/morpheus/_lib/cudf_helpers.pyx#L193

e.g. This API would allow us to create a `pylibcudf.Table` to pass into `cudf.DataFrame.from_pylibcudf`

Reply:
For that, I think we would use the existing `from_table_view` combined with `DataFrame.from_pylibcudf`, no?

Reply:
Yes, I think that would work, with the caveat that we don't do the specialized `column_owner` logic in that function. I don't think that's something that we really want to provide, though. The caller should know a priori if that is necessary and probably call `Column.from_column_view` in a loop over the columns of a `Table` if they are going that route.